src/pages/guidelines/type-pairing-and-spacing.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Type pairing and spacing
+description: Typography can help create clear hierarchies, organize information, and guide users through a product or experience. Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.
+---
+
+<PageDescription>
+
+Typography can help create clear hierarchies, organize information, and guide users through a product or experience. It's essential to understand the [typeface](https://www.carbondesignsystem.com/guidelines/typography/overview/#typeface:-ibm-plex), [type scale](https://www.carbondesignsystem.com/guidelines/typography/overview/#scale), [type tokens](https://www.carbondesignsystem.com/guidelines/typography/overview/#type-tokens-and-sets), and the [Expressive type sets](https://www.carbondesignsystem.com/guidelines/typography/expressive) before dive into creating an IBM.com website page.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Designing with type parings and spacings</AnchorLink>
+<AnchorLink>Support</AnchorLink>
+
+</AnchorLinks>
+
+## Designing with type parings and spacings
+
+Type parings and spacings support different levels of architecture or areas that require varying typographic hierarchies and are the keys to making the best IBM.com experiences through typography successfully.
+
+Below are some known type parings and spacings when designing for IBM.com to ensure you create clear hierarchies, proper information organization, and the best experience to guide users through the IBM.com experience.
```

Comment:
Should we rephrase to "Below are some common type pairings" ? Thinking that these are the ones we have built into the components, "known" just felt a little odd when I read it.

Reply:
Good point.